app/models/conversion_host.rb

```diff
@@ -1,13 +1,25 @@
 class ConversionHost < ApplicationRecord
   include NewWithTypeStiMixin
 
+  VALID_RESOURCE_TYPES = %w(VmOrTemplate Vm Host).freeze
```

Comment:
Vm is a subclass of VmOrTemplate so do you need to have both?

Reply:
Thought about that, but it doesn't seem to handle it.

Reply:
Polymorphic should set the type to the base class so if you're seeing VM I think you're doing something wrong...

Reply:
Upon further review, you are correct. I must've had something else broken when I test it earlier. Fixed.